### PR TITLE
Ask in PR template what parts of the code are modified

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Add a row in the PR template to ask what parts of the code are modified
| Type?         | improvement
| Category?     | PM
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22469)

### CODE REVIEW

**Comment by Progi1984** [comment]
What it the goal of this ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,5 +17,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Deprecations? | yes / no
 | Fixed ticket? | Fixes #{issue number here}.
 | How to test?  | Please indicate how to best verify that this PR is correct.
+| What are the parts of PrestaShop are modified this PR?  | Please indicate what parts of the software we need to verify to check everything is allright
```
Comment (SimonGrn) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
| What are the parts of PrestaShop modified by this PR?  | Please indicate what parts of the software we need to verify to check everything is alright
```

**Comment by SimonGrn** [comment]
> What it the goal of this ?

It's to help QA identifying what are the parts of PrestaShop we need to focus on. We need to use our exploratory testing time as efficiently as possible, so every bit of information is very important for us.
By making it in the template, we hope every contributor will think about it and help us.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (matks) [reply .github/PULL_REQUEST_TEMPLATE.md]: Waoh I am really tired... I write such bad English 😅 . Thanks for proofreading!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -17,5 +17,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Deprecations? | yes / no
 | Fixed ticket? | Fixes #{issue number here}.
 | How to test?  | Please indicate how to best verify that this PR is correct.
+| What are the parts of PrestaShop modified by this PR?  | Please indicate what parts of the software we need to verify to check everything is alright
```
Comment (jolelievre) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.
```

Comment (jolelievre) [reply .github/PULL_REQUEST_TEMPLATE.md]: A smaller column will be prettier I think?
(also a detail but the number of spaces will have to be adapted in other columns)

Comment (PululuK) [inline .github/PULL_REQUEST_TEMPLATE.md]: Duplicate of https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/pull-requests/#type--category ?

image

Comment (jolelievre) [reply .github/PULL_REQUEST_TEMPLATE.md]: Not exactly, the category is less precise This new field would be used to help QA team it's additional info to the `How to test?`

However the fact that you're asking the question means that it's not clear at all and that contributors won't understand its use 😅

Comment (PululuK) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sorry but I'm lost !

This PR suggest : `What are the parts of PrestaShop modified by this PR?`
The current doc : `The category is the main part of the project affected by your changes. Choose the code that most closely describes your change:`

What is the diff ?

Comment (matks) [reply .github/PULL_REQUEST_TEMPLATE.md]: @PululuK the category is actually used to build the Changelog and the Core Weekly. But "Back office" "or Web service" is very broad, it cannot help the QA.

The category aims to organize issues together while this new row aims to help QA know what to verify.

**Comment by matks** [comment]
@PululuK Here is an example of what we would like to have:


| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | I fixed TinyMCE responsive behavior in the BO now borders are OK (before it was broken)
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #829912
| How to test?  | Please see ticket
| What are the parts of PrestaShop modified by this PR?  | All the BO pages using TinyMCE: CMS Page, Product Page, Category Page

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review!

**Pull request merged**